shared/chat/conversation/attachment-fullscreen/index.desktop.tsx

```diff
@@ -1,53 +1,48 @@
 import * as React from 'react'
-import {
-  Box,
-  Box2,
-  Icon,
-  Text,
-  OrientedImage,
-  PopupDialog,
-  ProgressBar,
-  ProgressIndicator,
-  OverlayParentHOC,
-  OverlayParentProps,
-} from '../../../common-adapters'
+import * as Kb from '../../../common-adapters'
 import MessagePopup from '../messages/message-popup'
 import * as Styles from '../../../styles'
 import {Props} from '.'
-import KeyHandler from '../../../util/key-handler.desktop'
 
 type State = {
   loaded: string
   isZoomed: boolean
 }
 
-class _Fullscreen extends React.Component<Props & OverlayParentProps, State> {
-  _vidRef: {current: HTMLVideoElement | null} = React.createRef()
-  _mounted = false
+class _Fullscreen extends React.Component<Props & Kb.OverlayParentProps, State> {
   state = {isZoomed: false, loaded: ''}
-  _setLoaded = (path: string) => this.setState({loaded: path})
-  _isLoaded = () => this.props.path.length > 0 && this.props.path === this.state.loaded
+
+  private vidRef: {current: HTMLVideoElement | null} = React.createRef()
+  private mounted = false
+  private setLoaded = (path: string) => this.setState({loaded: path})
+  private isLoaded = () => this.props.path.length > 0 && this.props.path === this.state.loaded
+  private hotKeys = ['left', 'right']
+  private onHotKey = (cmd: string) => {
+    cmd === 'left' && this.props.onPreviousAttachment()
+    cmd === 'right' && this.props.onNextAttachment()
+  }
 
   componentDidMount() {
-    this._mounted = true
-    if (this._vidRef.current && this.props.autoPlay) {
-      this._vidRef.current.play()
+    this.mounted = true
+    if (this.vidRef.current && this.props.autoPlay) {
+      this.vidRef.current.play()
     }
   }
 
   componentWillUnmount() {
-    this._mounted = false
+    this.mounted = false
   }
 
   render() {
     return (
-      <PopupDialog onClose={this.props.onClose} fill={true}>
-        <Box style={styles.container}>
-          <Box style={styles.headerFooter}>
-            <Text type="BodySemibold" style={{color: Styles.globalColors.black, flex: 1}}>
+      <Kb.PopupDialog onClose={this.props.onClose} fill={true}>
+        <Kb.Box style={styles.container}>
+          <Kb.HotKey hotKeys={this.hotKeys} onHotKey={this.onHotKey} />
```

Comment:
feels weird to me to call this component `HotKey` rather than `HotKeyHandler` or something. Is this a pattern we use elsewhere?